confdgnmi/src/confd_gnmi_common.py

```diff
@@ -160,6 +160,19 @@ def make_formatted_path(gnmi_path, gnmi_prefix=None, quote_val=False) -> str:
     return path_str
 
 
+def prefix_gnmi_paths(path, prefix):
+    return gnmi_pb2.Path(elem=list(prefix.elem) + list(path.elem),
+                         origin=path.origin,
+                         target=path.target)
+
+
+def remove_path_prefix(path, prefix):
+    assert path.elem[:len(prefix.elem)] == prefix.elem[:]
```

Comment:
We were discussing that; if the assertion does not hold, there is a bug somewhere and there is hardly any point in returning any value. On the other hand, the comparison sometimes fails just because different prefixes used in the path...